drivers/bus.py

```diff
@@ -3,17 +3,23 @@
 """
 
 
-class Bus:
-    def __init__(self, logger):
+class BusHandler:
+    def __init__(self, logger, input_queue=None, out={}):
```

Comment:
I think we need input channels here as well. But that decision depends on a code that has not yet been written 😉